tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Setup "audit-test" test environment of a system with needed packages
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93441
+
+use base 'audit_test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration 'add_suseconnect_product';
+
+sub run {
+    my ($self)   = @_;
+    my $dir      = $audit_test::testdir;
+    my $file_tar = $audit_test::testfile_tar . ".tar";
```

Comment:
In general I suggest to always use single-ticks unless you want to expand variables within strings. This applies to all places where you use `"`

Reply:
Done.